zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/NotebookAuthorization.java

```diff
@@ -384,24 +385,30 @@ public boolean apply(NoteInfo input) {
   
   public void setNewNotePermissions(String noteId, AuthenticationInfo subject) {
     if (!AuthenticationInfo.isAnonymous(subject)) {
+      String admin = conf.getString(ConfVars.ZEPPELIN_OWNER_ROLE);
       if (isPublic()) {
         // add current user to owners - can be public
         Set<String> owners = getOwners(noteId);
         owners.add(subject.getUser());
+        owners.add(admin);
```

Comment:
I remember when I saw this PR first time, we don't have to add admin as the owner. Isn't it? Or am I missing something?

- https://github.com/apache/zeppelin/pull/2525/commits/1a1374ee4d93602b37577b96546ac80906cb7017

@yu74n

Reply:
@1ambda  Beginning of this PR, I was going to bypass authentication when a user login as admin, then we didn't have to add the admin user as the owner. But @felixcheung and @jongyoul suggested me that it would be better to add admin role when a notebook is created, after that I fixed this PR to do so... then we need to add admin as the owner, I think.

Reply:
IMO, it's not a scalable solution. For instance, if user wants to change the admin role name for some reason, they have to change the all existing owners of all notes.

Reply:
You are right, I didn't notice that. I'm trying to revert the code, but I think it would be easier to create new PR and cherry pick the code we need from this PR.

Reply:
I've created new PR #2585. Could you review the PR again please ?